fhir-parent/pom.xml

```diff
@@ -285,7 +285,7 @@
             <dependency>
                 <groupId>com.ibm.cos</groupId>
                 <artifactId>ibm-cos-java-sdk</artifactId>
-                <version>2.1.0</version>
+                <version>2.6.1</version>
```

Comment:
FYI, I looked at the dependency trees: 
 +- com.ibm.cos:ibm-cos-java-sdk:jar:2.1.0:compile
[INFO] |  +- com.ibm.cos:ibm-cos-java-sdk-s3:jar:2.1.0:compile
[INFO] |  |  +- (com.ibm.cos:ibm-cos-java-sdk-kms:jar:2.1.0:compile - omitted for duplicate)
[INFO] |  |  \- (com.ibm.cos:ibm-cos-java-sdk-core:jar:2.1.0:compile - omitted for duplicate)
[INFO] |  +- com.ibm.cos:ibm-cos-java-sdk-kms:jar:2.1.0:compile
[INFO] |  |  \- (com.ibm.cos:ibm-cos-java-sdk-core:jar:2.1.0:compile - omitted for duplicate)
[INFO] |  \- com.ibm.cos:ibm-cos-java-sdk-core:jar:2.1.0:compile
[INFO] |     +- (commons-logging:commons-logging:jar:1.1.3:compile - omitted for conflict with 1.2)
[INFO] |     +- (org.apache.httpcomponents:httpclient:jar:4.5.6:compile - version managed from 4.5.2; omitted for duplicate)
[INFO] |     +- software.amazon.ion:ion-java:jar:1.0.2:compile
[INFO] |     +- com.fasterxml.jackson.core:jackson-databind:jar:2.6.6:compile
[INFO] |     |  +- com.fasterxml.jackson.core:jackson-annotations:jar:2.6.0:compile
[INFO] |     |  \- com.fasterxml.jackson.core:jackson-core:jar:2.6.6:compile
[INFO] |     +- com.fasterxml.jackson.dataformat:jackson-dataformat-cbor:jar:2.6.6:compile
[INFO] |     |  \- (com.fasterxml.jackson.core:jackson-core:jar:2.6.6:compile - omitted for duplicate)
[INFO] |     +- (org.apache.httpcomponents:httpasyncclient:jar:4.1.3:compile - omitted for conflict with 4.1.4)
[INFO] |     \- joda-time:joda-time:jar:2.8.1:compile

+- com.ibm.cos:ibm-cos-java-sdk:jar:2.6.1:compile
[INFO] |  +- com.ibm.cos:ibm-cos-java-sdk-s3:jar:2.6.1:compile
[INFO] |  |  +- (com.ibm.cos:ibm-cos-java-sdk-kms:jar:2.6.1:compile - omitted for duplicate)
[INFO] |  |  \- (com.ibm.cos:ibm-cos-java-sdk-core:jar:2.6.1:compile - omitted for duplicate)
[INFO] |  +- com.ibm.cos:ibm-cos-java-sdk-kms:jar:2.6.1:compile
[INFO] |  |  \- (com.ibm.cos:ibm-cos-java-sdk-core:jar:2.6.1:compile - omitted for duplicate)
[INFO] |  \- com.ibm.cos:ibm-cos-java-sdk-core:jar:2.6.1:compile
[INFO] |     +- (commons-logging:commons-logging:jar:1.1.3:compile - omitted for conflict with 1.2)
[INFO] |     +- (org.apache.httpcomponents:httpclient:jar:4.5.6:compile - version managed from 4.5.9; omitted for duplicate)
[INFO] |     +- software.amazon.ion:ion-java:jar:1.2.0:compile
[INFO] |     +- com.fasterxml.jackson.core:jackson-databind:jar:2.10.2:compile
[INFO] |     |  +- com.fasterxml.jackson.core:jackson-annotations:jar:2.10.2:compile
[INFO] |     |  \- com.fasterxml.jackson.core:jackson-core:jar:2.10.2:compile
[INFO] |     +- com.fasterxml.jackson.dataformat:jackson-dataformat-cbor:jar:2.10.0:compile
[INFO] |     |  \- (com.fasterxml.jackson.core:jackson-core:jar:2.10.0:compile - omitted for conflict with 2.10.2)
[INFO] |     \- joda-time:joda-time:jar:2.8.2:compile

Reply:
reasonably the same and up to date.

Reply:
thanks for the check!